guides/imgix-guide/index.mdx

```diff
@@ -0,0 +1,29 @@
+---
+_schema: default
+_uuid: 01ecefe3-3841-4d5d-aacc-450959563c57
+_created_at: 2024-05-31T15:53:59+12:00
+title: Introduction
+nav_title: Introduction
+order: 1
+published: true
+image: >-
+  https://cc-dam.imgix.net/documentation/images/default-thumbnail/2023-Q4/CloudCannon-Documentation-Thumbnail.png
+description: >-
+  Learn how to use the CloudCannon IMGIX integration with an Amazon S3 digital
```

Comment:
Seems like imgix mostly use all lowercase when talking about themselves - do we perhaps want to match that?

Reply:
I'll correct this :)